project/ProjectPlugin.scala

```diff
@@ -79,6 +79,15 @@ object ProjectPlugin extends AutoPlugin {
       )
     )
 
+    lazy val clientCacheSettings: Seq[Def.Setting[_]] = Seq(
+      libraryDependencies := Seq(
+        %%("log4s", V.log4s),
+        %%("fs2-core", "0.10.1"),
```

Comment:
This it the fs2 version used by the fs2-reactive-streams version that we are currently using.
This one is sadly still on cats-effect 0.8.0

Reply:
Updated to 0.10.5, also added an explicit dependency to fs2 0.10.5 to `internal` so that the transitive fs2 0.10.1 gets evicted.

Reply:
Cool, thanks @peterneyens